docs/future-work/learning-module-improvements/use-models-with-less-points.md

```diff
@@ -1,3 +1,5 @@
 ---
 title: Use Models with Less Points
```

Comment:
I hate to be that guy, but "Fewer" is also an option here.

Reply:
Haha fair point, thanks